### ISSUE TITLE
ModuleNotFoundError: No module named pkg_resources

### ISSUE THREAD

**Issue by tnguyen14** [googleapis/python-monitoring#322]
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

Please run down the following list and make sure you've tried the usual "quick fixes":

  - Search the issues already opened: https://github.com/googleapis/python-monitoring/issues
  - Search StackOverflow: https://stackoverflow.com/questions/tagged/google-cloud-platform+python

If you are still having issues, please be sure to include as much information as possible:

#### Environment details

  - OS type and version: linux
  - Python version: 3.8.13
  - pip version: `pip --version`
  - `google-cloud-monitoring` version: 2.9.1

#### Stack trace
```
Python 3.8.13 (default, Mar 18 2022, 13:52:37)
[GCC 9.3.1 20200408 (Red Hat 9.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from google.cloud import monitoring_v3
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../dependencies/site-packages/google/cloud/monitoring_v3/__init__.py", line 17, in <module>
    from .services.alert_policy_service import AlertPolicyServiceClient
  File "/.../dependencies/site-packages/google/cloud/monitoring_v3/services/alert_policy_service/__init__.py", line 16, in <module>
    from .client import AlertPolicyServiceClient
  File "/.../dependencies/site-packages/google/cloud/monitoring_v3/services/alert_policy_service/client.py", line 20, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

This library depends on `setuptools`, which was included in `google-api-core` up to `2.4.0`, but was removed in `2.5.0` https://github.com/googleapis/python-api-core/pull/339

**Comment by parthea** [comment]
Hi @tnguyen14 ,

Thanks for reporting this issue and proposing a fix! I'm going to check if we can remove the dependency on setuptools.

I've also opened an issue for a long term fix: https://github.com/googleapis/gapic-generator-python/issues/1316

**Comment by parthea** [comment]
Closed via eaaca4815872d78725893b0aa26ffd96d84d58d5
